rcongui/src/components/Blacklist/BlacklistListCreateDialog.js

```diff
@@ -49,7 +49,7 @@ export default function BlacklistListCreateDialog({
   }
 
   function toggleAllServers(enabled) {
-    setServerNumbers(enabled ? null : Object.keys(servers).map(parseInt));
+    setServerNumbers(enabled ? null : Object.keys(servers).map(n => parseInt(n)));
```

Comment:
As far as I know, this makes no difference.

Reply:
It definitely did. The problem is that the function passed to `map` takes up to three arguments, and will toss as many as it can fit inside of `parseInt`, which we do not want and was causing `['1', '2', '3']` to be mapped to `[1, null, null]`.

Reply:
Oh. Good point.